tailwind custom color name change

theShinigami opened the pull request:
Updated the custom color names for the following: 

brick-red -> red
lucky-orange -> orange
ice-cream-purple -> purple
centeral-park-green -> green
carolina-yellow -> yellow

#10

tim-schilling reviewed (approved):
Ugh, copying and pasting is hard.

theShinigami committed:
tailwind custom color name change
changed custom color names

brick-red -> red
lucky-ornage -> orange
ice-cream-purple -> purple
centeral-park-green -> green
carolina-yellow -> yellow

mtrythall commented:
Thanks for this!
